tools/pontoon/ui.md

```diff
@@ -0,0 +1,88 @@
+# Understanding the Pontoon User Interface
+
+## Pontoon Translation Workspace
+
+Pontoon's translation workspace features the projects string list in the sidebar on the left and the editing space on the right. The features in the Pontoon user interface are labeled in the image below:
+
+![Translation Workspace](/assets/images/pontoon/ui/translation_workspace.png)
+
+### Menu Bar
+
+The Menu Bar allows you to navigate between projects without leaving the Translation Workspace.
+
+![Menu bar](/assets/images/pontoon/ui/menu_bar.png)
+
+To navigate to another project, click on each item to open the drop-down menu and select the locale, project, and resource to work on. Then click `Go`.
+
+![Navigating in the Menu bar](/assets/images/pontoon/ui/menu_nav.png)
+
+*Note that clicking* `All` *or* `Current` *in the drop-down menu will navigate away from the Translation Workspace.*
+
+A string overview is located to the right of the Menu Bar. Translators can view the overview in detail by clicking it once.
+
+![Expanded String Overview](/assets/images/pontoon/ui/string_overview.png)
+
+A project information icon is located to the right of the string overview. Translators can view information regarding the project, it's priority level, and testing by clicking the icon.
+
+### Sidebar
+
+The sidebar displays the list of strings in the current project resource. Each string is displayed with the string status (i.e. *Missing*, *Translated*, etc.) marked by a colored square, the source string, and the translation or suggestion if available.
```

Comment:
in the selected resource for the current project
translation or suggestion -> approved translation or most recent suggestion